mlapi/model/facet.py

```diff
@@ -7,6 +7,12 @@ def __init__(self, name, value):
         self.name = name
         self.value = value
 
+    def __eq__(self, other):
+        return self.name == other.name and self.value == other.value
```

Comment:
Detail (*1000): There should be a check to see whether other is an instance of Facet to  avoid comparison with other objects that have a name field